micro-airport-location/redis-deployment.yml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: redis-v1
+  namespace: default
+  labels:
+    app: redis
+    version: v1
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: redis
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: redis
+        version: v1
+    spec:
+      containers:
+        - name: redis
+          image: redis:6.0
+          imagePullPolicy: IfNotPresent
```

Comment:
@cichan02 should be `Always`
we expect our pod to be crashed each time the container not found in the registry